src/main/java/DataModel/ExchangeRateDTO.java

```diff
@@ -1,19 +1,44 @@
 package DataModel;
 
+import Mappers.JSONMapper;
 import services.FetchToTheForexAPI;
 
-import java.sql.SQLOutput;
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.HashMap;
 import java.util.Objects;
 
 public class ExchangeRateDTO {
```

Comment:
you have ExchangeRate and ExchangeRateDTO.
What is the function of these classes?

Reply:
These classes separate the business and technical layers?